### ISSUE TITLE
DwC archives -> GEE 

### ISSUE THREAD

**Issue by 7yl4r** [USF-IMARS/obis_notebooks#2]
I have done some work to get DwC archives into google earth engine. One use-case here is to use GEE's built-in species distribution modeling functions.

Below is an example SDM @luislizcano built using seagrass data.

![image](https://user-images.githubusercontent.com/1051390/140542919-93468681-9034-489b-8111-4cbc0bce8ca9.png)

He was able to put that together quickly and I think there is a lot of potential there.

I grabbed occurrences of Red Mangroves in the Caribbean from GBIF and was able to import the occurrences.txt into GEE:

![image](https://user-images.githubusercontent.com/1051390/140543216-6bcad338-94ec-4243-9907-f7f1303e3e4e.png)

Next up I want to color points based on a property in the `ee.Feature`. I found [two ways to do that](https://gis.stackexchange.com/a/269949/107752) but I haven't got either of them to work for me yet. 

In order to put this into an SDM I think we also need to add some `absence` points, since all these are `presence` only.

**Comment by luislizcano** [comment]
Yes, you need another dataset of points representing absence. I would distribute the absence points depending on what the input variables are, e.g. if DEM data is available, mangrove absence points would be placed at very high elevations or inland places.
